dts/arm/nordic/nrf9160_common.dtsi

```diff
@@ -161,6 +154,13 @@ gpio0: gpio@842500 {
 	status = "disabled";
 };
 
+rtc0: rtc@14000 {
```

Comment:
Why was this needed?

Reply:
Also, no alias is defined for this one....

Reply:
> Why was this needed?

To get the base address of this peripheral in [nrfx_config_nrf9160.h](https://github.com/zephyrproject-rtos/zephyr/pull/12531/files#diff-7c92595657f7db412148884cfa771899R80).

> Also, no alias is defined for this one....

Right. This is missing. I'll add the alias.

Reply:
Done.